content/en/docs/tasks/observability/metrics/customize-metrics/index.md

```diff
@@ -81,6 +81,7 @@ the example application throughout this task.
     {{< /tip >}}
 
     {{< text yaml >}}
+    # custom_metrics.yaml
```

Comment:
I think the usual format followed is similar to https://github.com/istio/istio.io/blame/master/content/en/docs/tasks/observability/distributed-tracing/mesh-and-proxy-config/index.md#L75